dags/pytorch_xla/configs/pytorchxla_torchbench_config.py

```diff
@@ -14,15 +14,71 @@
 
 """Utilities to construct configs for pytorchxla_torchbench DAG."""
 
-import datetime
+import enum
 from typing import Tuple
 from xlml.apis import gcp_config, metric_config, task, test_config
 import dags.vm_resource as resource
 from dags import gcs_bucket, test_owner
 
 
-def set_up_torchbench_tpu(model_name: str = "") -> Tuple[str]:
+class VERSION(enum.Enum):
+  NIGHTLY = enum.auto()
+  R2_2 = enum.auto()
+  R2_3 = enum.auto()
+
+
+class VERSION_MAPPING:
+
+  class NIGHTLY(enum.Enum):
+    TORCH_XLA_TPU_WHEEL = "https://storage.googleapis.com/pytorch-xla-releases/wheels/tpuvm/torch_xla-nightly-cp310-cp310-linux_x86_64.whl"
+    TORCH_XLA_CUDA_WHEEL = "https://storage.googleapis.com/pytorch-xla-releases/wheels/cuda/12.1/torch_xla-nightly-cp38-cp38-linux_x86_64.whl"
+    TORCH = "torch"
+    TORCHVISION = "torchvision"
+    TORCHAUDIO = "torchaudio"
+    TORCH_XLA_GPU_DOCKER = "us-central1-docker.pkg.dev/tpu-pytorch-releases/docker/xla:nightly_3.10_cuda_12.1"
+    TORCH_INDEX_CPU_URL = "https://download.pytorch.org/whl/nightly/cpu"
+    TORCH_INDEX_CUDA_URL = "https://download.pytorch.org/whl/nightly/cu121"
+    TORCH_REPO_BRANCH = "-b main"
+    TORCH_XLA_REPO_BRANCH = "-b master"
+
+  class R2_2(enum.Enum):
+    TORCH_XLA_TPU_WHEEL = "https://storage.googleapis.com/pytorch-xla-releases/wheels/tpuvm/torch_xla-2.2.0-cp310-cp310-manylinux_2_28_x86_64.whl"
+    TORCH_XLA_CUDA_WHEEL = "https://storage.googleapis.com/pytorch-xla-releases/wheels/cuda/12.1/torch_xla-2.2.0-cp310-cp310-manylinux_2_28_x86_64.whl"
+    TORCH = "torch==2.2.0"
+    TORCHVISION = "torchvision==0.17.0"
+    TORCHAUDIO = "torchaudio==2.2.0"
+    TORCH_XLA_GPU_DOCKER = "us-central1-docker.pkg.dev/tpu-pytorch-releases/docker/xla:r2.2.0_3.10_cuda_12.1"
+    TORCH_INDEX_CPU_URL = "https://download.pytorch.org/whl/cpu"
+    TORCH_INDEX_CUDA_URL = "https://download.pytorch.org/whl/cu121"
+    TORCH_REPO_BRANCH = "-b v2.2.0"
+    TORCH_XLA_REPO_BRANCH = "-b v2.2.0"
+
+  # TODO(@siyuan): Please update the 2.3 rc to the latest.
+  class R2_3(enum.Enum):
+    TORCH_XLA_TPU_WHEEL = "https://storage.googleapis.com/pytorch-xla-releases/wheels/tpuvm/torch_xla-2.3.0rc2-cp310-cp310-linux_x86_64.whl"
+    TORCH_XLA_CUDA_WHEEL = "https://storage.googleapis.com/pytorch-xla-releases/wheels/cuda/12.1/torch_xla-2.3.0rc2-cp310-cp310-linux_x86_64.whl"
+    TORCH = "torch==2.3.0"
```

Comment:
does 2.3.0 exist now for torch?

Reply:
Yes, if you use `pip install --pre torch==2.3.0 https://download.pytorch.org/whl/test/cpu`, it will install the latest torch 2.3.0 rc build.